server/common/dao/src/main/java/org/kaaproject/kaa/server/common/dao/DaoConstants.java

```diff
@@ -419,6 +422,12 @@
     public static final String OPT_LOCK = "opt_lock";
     public static final String APPLIED = "[applied]";
 
+    public static final BiMap<Character, Character> CODE_CHARACTERS = HashBiMap.create();
```

Comment:
This is specific to Mongo endpoint profile implementation. Please move this to corresponding class and make it private. Constant should be names RESERVED_CHARACTERS